### ISSUE TITLE
Accessibility: Aria labels

### ISSUE THREAD

**Issue by melissamcewen** [RachelleEAD/bwui-capstone-placeholder#2]
Nice Aria labels, there were just a few things that seemed to need some improvement. I use this Chrome Extension called Wave to do a report and it said
https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US

> An aria-labelledby or aria-describedby reference exists, but the target for the reference does not exist.

The MDN reference on this is good https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

It basically ties an input to a label, but there is no label here. The `aria-label` attribute is the best choice for forms without a label https://www.w3.org/WAI/tutorials/forms/labels/#hiding-label-text

Or you can create a label and hide it with CSS
```html
      <div class="search-bar" >
        <span for="search" id="search-bar-label" class="visuallyhidden">Search: </span>
        <input type="text" name="search" placeholder="Search for items" aria-labelledby="search-bar-label">
        <a href="" class="icon"><img src="resources/img/search.svg"></a>
      </div>
```

`visuallyhidden` should look something like this https://css-tricks.com/places-its-tempting-to-use-display-none-but-dont/

I often had to do this approach for sites where I had less control over the form code
